lib/_utils_multiple-choice.ts

```diff
@@ -0,0 +1,41 @@
+import { OptionType } from "../components/predict-form/PredictProvider"
+
+export function normalizeOptionsToHundred(options: OptionType[]) {
+  const optionsCopy = [...options]
+  const optionsWithForecasts = optionsCopy.filter(
+    ({ forecast }) => forecast !== undefined && !Number.isNaN(forecast),
+  ) as Array<OptionType & { forecast: number }>
```

Comment:
Typing forecast here means that I don't have to assert it to be a value in each of my filters

Reply:
Alternatively, I see in PredictProvider.tsx that optionSchema can be NaN. Any reason why? If we could get rid of that then we could at least get rid of the isNaN check – not urgent though